src/styles.css

```diff
@@ -1,4 +1,101 @@
-/* You can add global styles to this file, and also import other style files */
+html,
+body {
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
 
-html, body { height: 100%; }
-body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
+body {
+  background-image: url("https://images.pexels.com/photos/1422770/pexels-photo-1422770.png?auto=compress&cs=tinysrgb&dpr=3&h=750&w=1260");
```

Comment:
artifacts need to be provided locally so we don't depend on an external url to be available. you can download the image and place it in the assets img directory